platform/mikro-e/Readme.md

```diff
@@ -29,8 +29,8 @@ environment variables and compile an example, say "Hello World" which is include
 
     $ export PATH=$PATH:/opt/microchip/xc32/v1.34/bin/
 
-* Platform mikro-e makefile builds the application for ca8210 by default, a cc2520 build can be enabled by passing USE_CC2520=1.
-* 6lowpan channel(default 26) and pan_id(default 0xabcd) can be passed from makefile options.
+* Platform mikro-e makefile builds the application for ca8210.
```

Comment:
Do we even need to mention CA8210 in the readme now ? Since we have only one type of build, what about 
```Platform mikro-e makefile builds applications for the clicker.```